### ISSUE TITLE
bref.dev not working: Uncaught Error: Class 'GuzzleHttp\Client' not found

### ISSUE THREAD

**Issue by khromov** [brefphp/bref#768]
Getting the issue below. Based on `package.lock`, it doesn't seem like Guzzle gets installed? It's only listed as a `require-dev` dependency.

```
/GitHub/wps$ vendor/bin/bref bref.dev
Creating a short URL for https://xxx.execute-api.eu-central-1.amazonaws.com/dev
PHP Fatal error:  Uncaught Error: Class 'GuzzleHttp\Client' not found in /home/k/GitHub/wps/vendor/bref/bref/bref:422
Stack trace:
#0 [internal function]: Silly\Application->{closure}()
#1 /home/k/GitHub/wps/vendor/php-di/invoker/src/Invoker.php(83): call_user_func_array()
#2 /home/k/GitHub/wps/vendor/mnapoli/silly/src/Application.php(95): Invoker\Invoker->call()
#3 /home/k/GitHub/wps/vendor/symfony/console/Command/Command.php(256): Silly\Application->Silly\{closure}()
#4 /home/k/GitHub/wps/vendor/symfony/console/Application.php(920): Symfony\Component\Console\Command\Command->run()
#5 /home/k/GitHub/wps/vendor/symfony/console/Application.php(266): Symfony\Component\Console\Application->doRunCommand()
#6 /home/k/GitHub/wps/vendor/symfony/console/Application.php(142): Symfony\Component\Console\Application->doRun()
#7 /home/k/GitHub/wps/vendor/bref/bref/bref(460): Symfony\Component\Console\Application->run()
#8 {main}
  thrown in /home/k/GitHub/wps/vendor/bref/bref/bref on line 422
```

**Comment by khromov** [comment]
Solved by manually running `composer require guzzlehttp/guzzle`

**Comment by mnapoli** [comment]
Closed by #796
